### ISSUE TITLE
ctrl + z doesn't cancel the last action

### ISSUE THREAD

**Issue by Kaiz3rTool** [wonderunit/storyboarder#1250]
on shots with metadata, when i control Z, the last drawing action isnt cancelled 
it affect only the text in dialogue/action/note tab

this problem appear only on images with metadata

**Comment by denisvanw** [comment]
In my case, the problem exists even on boards without any metadata...

**Comment by audionerd** [comment]
For some actions, undo is adding two undo steps instead of one, so you have to press it twice to actually undo what you want. Is this the bug you're running into?

**Comment by Kaiz3rTool** [comment]
No, no matter how many times i press ctrl Z, it doesnt cancel drawing operation if metadata are on the shot

if i manually clic on undo, on the arrow, no problem at all, it cancel the right things

**Comment by denisvanw** [comment]
Same for me. No undo with CTRL-Z no matter how many times I press the key. 
But cliking the "undo" icon works fine.
